src/components/calcite-notice/calcite-notice.tsx

```diff
@@ -95,6 +95,15 @@ export class CalciteNotice {
     if (!widths.includes(this.width)) this.width = "auto";
   }
 
+  componentDidLoad() {
+    this.noticeLinkSlot = this.el.shadowRoot.querySelector(
```

Comment:
Does the polyfill handle querying shadowRoot? Might be good to test with edge.

Reply:
Good catch, changed this to not use shadowRoot, tested and works in Edge.